Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.22-alpine3.19 AS builder
```

Comment:
FYI 3.21 is available.

Reply:
Thanks, but just keeping up to date with upstream rather than getting ahead https://github.com/golang-migrate/migrate/blob/d477553dcd9ae33e2698c09f186e988767d71b79/Dockerfile#L1